Traceback (most recent call last):
  File "tools/analysis_tools/analyze_logs.py", line 8, in <module>
    import seaborn as sns
ModuleNotFoundError: No module named 'seaborn'
